client/src/Components/Auth/ResetPassword/ResetPassword.js

```diff
@@ -0,0 +1,359 @@
+import React, { useEffect, useState } from 'react';
+import dotenv from 'dotenv';
+import { useDispatch } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import axios from 'axios';
+import Bounce from 'react-activity/dist/Bounce';
+import 'react-activity/dist/Bounce.css';
+import Loader from 'react-loader-advanced';
+import { ToastContainer, toast } from 'react-toastify';
+const initialState = {
+  password: '',
+  confirmPassword: '',
+};
+
+dotenv.config();
+function LoginPage(props) {
+  const history = useHistory();
+  const dispatch = useDispatch();
+
+  const [isLogin, setIsLogin] = useState(false);
+  const [showPassword, setShowPassword] = useState(false);
+  const [formData, setFormData] = useState(initialState);
+  const [redComp, setRedComp] = useState(false);
+  const [loader, setLoader] = useState(false);
+  const [errorHandler, setErrorHandler] = useState(false);
+  useEffect(() => {
+    // console.log(redComp)
+    if (redComp) {
+      props.history.push('/room');
+      window.location.reload();
+    }
+  }, [redComp]);
+  const submitHandler = async (e) => {
+    e.preventDefault();
+    const { id: JWT_KEY } = props.match.params;
+    const { password, confirmPassword } = formData;
+    if (password !== confirmPassword) {
+      setErrorHandler('Password doesn"t match');
+    } else {
+      try {
+        await axios.post(`/api/_reset_password/${JWT_KEY}`);
```

Comment:
PassParametes password and confirm password, Also modify back-end route get to post.